relaton-bib.gemspec

```diff
@@ -36,4 +36,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "bibtex-ruby"
   spec.add_dependency "iso639"
   spec.add_dependency "nokogiri"
+  spec.add_dependency "rexml"
```

Comment:
That's very strange -- we don't use REXML...?

Reply:
We do. Or more precisely, BibTeX-Ruby uses it (see https://github.com/inukshuk/bibtex-ruby/pull/146).

Ruby Standard Library is in the process of moving into gems. REXML has been degraded to a bundled gem in Ruby 3.0, 
REXML is part of Ruby Standard Library. However, some time ago they started to move all the stdlib to gems (see https://stdgems.org/). Since Ruby 3.0, REXML has been degraded to a bundled gem, therefore has to be specified as dependency (see https://stdgems.org/rexml/).

Reply:
We don't write out to BibTeXML though, so that functionality is not needed here.

Reply:
Indeed. Initially it wasn't working without REXML, but they fixed it in https://github.com/inukshuk/bibtex-ruby/commit/4308f93331bc8c20c74f53d582e8ad8dcf6dc420. I'll remove this commit from the pull request.